### PR TITLE
[Refactor] add RawSendBytes, SerializedBytes and CompressedBytes for exchange sink

### PR DESCRIPTION
**Pull request by fzhedu**
RawSendBytes: origional bytes of chunks to be sent;
SerializedBytes: size of chunks afer serialized, may be encoded for integer and binary columns;
CompressedBytes: size of chunks after compressed.

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [x] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

File: be/src/exec/pipeline/exchange/exchange_sink_operator.cpp [hunk 1]
```diff
@@ -665,6 +667,8 @@ void ExchangeSinkOperator::close(RuntimeState* state) {
 
 Status ExchangeSinkOperator::serialize_chunk(const Chunk* src, ChunkPB* dst, bool* is_first_chunk, int num_receivers) {
     VLOG_ROW << "[ExchangeSinkOperator] serializing " << src->num_rows() << " rows";
+    auto raw_send_bytes = serde::ProtobufChunkSerde::max_serialized_size(*src, nullptr);
+    COUNTER_UPDATE(_send_bytes_counter, raw_send_bytes * num_receivers);
```
Comment (silverbullet233) [inline be/src/exec/pipeline/exchange/exchange_sink_operator.cpp:671]: max_serialized_size is not an accurate value, shoule we use src->bytes_usage or src->memory_usage?

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:disappointed: **fail** : 0 / 14 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/exec/pipeline/exchange/exchange_sink_operator.cpp|0|14|00.00%|[442, 443, 444, 670, 671, 694, 695, 697, 703, 709, 712, 725, 730, 731]|

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:disappointed: **fail** : 0 / 14 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/exec/pipeline/exchange/exchange_sink_operator.cpp|0|14|00.00%|[442, 443, 444, 670, 671, 694, 695, 697, 703, 709, 712, 725, 730, 731]|

**Review by silverbullet233 — APPROVED** [review]

File: be/src/exec/pipeline/exchange/exchange_sink_operator.cpp (hunk 1 above)
Comment (fzhedu) [reply be/src/exec/pipeline/exchange/exchange_sink_operator.cpp]: not really, max_serialized_size is closer the accurate value.

**Review by liuyehcf — APPROVED** [review]

**Comment by fzhedu** [comment]
@mergify backport abtest

**Comment by fzhedu** [comment]
@mergify backport branch-3.1

**Comment by fzhedu** [comment]
@mergify backport branch-3.0

**Comment by fzhedu** [comment]
@mergify backport branch-3.2

**Pull request merged**